docs/tutorials/index.rst

```diff
@@ -0,0 +1,7 @@
+Tutorials
+============
+
+Tutorials live here, except for the basic Quick Start tutorial.
```

Comment:
Why the exception, re: quick start? 
It could live here and be linked directly from the front page.

Or is the quick start intrinsically not a tutorial?  In that case we shouldn't mention it.

Reply:
Mostly: Much smaller scope, also, I want it in the table of contents on the main page, which unless someone else knows how to do magic like that, I think means it needs to live a level up.

Reply:
The sphinx toctree is actually one thing, spread out over multiple files.

You can see it in the current docs by looking at the current index, source and rendered. Both inline headings and subpages are included in the TOC.